### PR TITLE
GraphNG: Minor polish & updates to new time series panel and move it from alpha to beta

### PR DESCRIPTION
**Pull request by torkelo**
* [x] Updates name to `Time series' (white space)
* [x] Updates sort to 2 so it's after the standard graph panel
* [x] Updates default fill to zero
* [x] Fixes default gradient mode 
* [x] Adds beta badge to the panel in viz picker (and changes timeseries panel from alpha to beta)

### CODE REVIEW

File: packages/grafana-data/src/types/plugin.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { LiveChannelSupport } from './live';
 export enum PluginState {
   alpha = 'alpha', // Only included if `enable_alpha` config option is true
   beta = 'beta', // Will show a warning banner
+  stable = 'stable', // Will show a warning banner
```
Comment (ryantxu) [inline packages/grafana-data/src/types/plugin.ts:9]: Will not show?

**Review by ryantxu — APPROVED** [review]

File: packages/grafana-data/src/types/plugin.ts (hunk 1 above)
Comment (torkelo) [reply packages/grafana-data/src/types/plugin.ts:9]: yepp, will not show. Just in case someone put that in plugin.json

Comment (leeoniya) [reply packages/grafana-data/src/types/plugin.ts:9]: @torkelo i think he meant that you missed "not" in the comment :)

Comment (ryantxu) [reply packages/grafana-data/src/types/plugin.ts:9]: yup -- was on my phone so the suggestion button was awkward ;)

Comment (torkelo) [reply packages/grafana-data/src/types/plugin.ts:9]: Oops :) 
https://github.com/grafana/grafana/commit/7ca66b204004d4d0089de652cd47c51d4195cf36

**Pull request merged**